docs/accordproject-faq.md

```diff
@@ -17,42 +17,41 @@ While the Accord Project technology is targeted at the development of smart lega
+Contractual agreements sit at the heart of any organization, governing relationships with employees, shareholders, customers, suppliers, financiers, and more. Yet contracts today are not capable of being efficiently managed as the valuable assets they are. Currently contracts exist as static text documents stored in cloud storage services, dated contract management systems, or even email inboxes. Often these documents are Word files or PDFs that can only be interpreted by humans. A smart legal contract, by contrast, can be interpreted by machines.
 
-Smart Legal Contracts can be easily searched, analyzed, queried, and understood. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
+Smart Legal Contracts can be easily searched, analyzed, queried, and understood even by machines being static-typed. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
 
 The data model is used to ensure that all of the necessary data is present in the contract, and that this data is valid. In addition, it provides the necessary structure to enable contracts to "come alive" by adding executable logic.
 
 The result is a contract that is:
 
 
-* Searchable 
-* Analyzable 
-* Real-time 
-* Integrated 
+* Searchable
+* Analyzable
+* Real-time
+* Integrated
 
-Consequently, contracts are transformed from business liabilities in constant need of management to assets capable of providing real business intelligence and value. A Smart Contract contains a data model so that the data is part of the contract and not something held in an external system. The logical operations of the contract are also part of the contract. The contract can update itself and react to the outside world. Rather than being stored in filing cabinet it is a living breathing process.
+Consequently, contracts are transformed from business liabilities in constant need of management to assets capable of providing real business intelligence and value. A Smart Contract contains a data model so that the data is part of the contract and not something held in an external system. The logical operations of the contract are also part of the contract. The contract can update itself and react to the outside world. Rather than being stored in filing cabinet, it is a constantly updating dynamic process.
```

Comment:
I think `living breathing` should not be changed.

Reply:
Understood.